actions/send_email.py

```diff
@@ -8,7 +8,7 @@
 
 
 class SendEmail(Action):
-    def run(self, email_from, email_to, subject, message, account, mime="plain", attachments=None):
+    def run(self, email_from, email_to, subject, message, account, email_cc=[], mime="plain", attachments=None):
```

Comment:
This doesn't do what you think it does, and it introduces a very frustrating subtle bug.

See this [Stack Overflow post](https://stackoverflow.com/questions/366422/what-is-the-pythonic-way-to-avoid-default-parameters-that-are-empty-lists) for more about it, and how to fix it.

Reply:
Thanks, will do!

Reply:
Better now?